docs/contributing.md

```diff
@@ -0,0 +1,151 @@
+you may include a note in the pull request acknowledging that it is incomplete
+along with a request for feedback.
+
+Pull requests will generally not be accepted if any tests are failing.
+Therefore, it is recommended that you run the tests before submitting your pull
+request, ideally using `tox` in order to check that your modifications are
+compatible with all supported versions of Python. After making a pull request,
+check the Travis build status in the GitHub interface to ensure that all tests
+are running as expected.
+
+## Versions
+
+Python-Markdown follows [Semantic Versioning] and uses the
+`MAJOR.MINOR.POINT(PRERELEASE/DEV)` format for specifying releases. The status
+of the `master` branch should always be identified in the `__version_info__`
+variable defined in [markdown/__init__.py] and should conform to [PEP 440].
+
+### Version Status
+
+A MAJOR version is in dev status when the MINOR version is `0`, the POINT
+version is `0`, and the version includes a DEV segment.
+
+A MINOR version is in dev status when the MINOR version is not `0`, the POINT
+version is `0`, and the version includes a DEV segment.
+
+At all other times, the code is considered stable and release-ready.
+
+MAJOR and MINOR releases may or may not get pre-releases (beta, release
+candidate, etc.) at the discretion of the project maintainers.
+
+### Version Workflow
+
+Bug fixes may be committed to the `master` branch at any time.
+
+New features and backward incompatible changes may only be committed to the
+`master` branch when the MAJOR and/or MINOR version is in `dev` status.
+
+A separate commit to the `master` branch should be made to bump up the MAJOR
+and/or MINOR version and set the `dev` status. Only then will any PRs
+implementing new features or backward incompatible changes be accepted.
+
+If a bug fix is deemed to be important and the `master` branch is in `dev`
+status, a backport of the fix should be committed to a version branch. If the
+appropriate version branch does not exist, then it should be created and a PR
+back-porting the fix made against that branch.The version branch should be named
+with the most recently released MINOR version. For example, if the `master`
+branch is at `3.1.0dev` and the most recent MINOR release was `3.0.4`, then the
+version branch would be named `3.0` and any releases from that branch would
+increment the POINT version only (`3.0.5`, `3.0.6`...).
+
+
+[Python-Markdown Organization]: https://github.com/Python-Markdown
+[Python-Markdown Code of Conduct]: https://github.com/Python-Markdown/markdown/blob/master/CODE_OF_CONDUCT.md
+[Python-Markdown/markdown]: https://github.com/Python-Markdown/markdown
+[issue tracker]: https://github.com/Python-Markdown/markdown/issues
+[syntax rules]: http://daringfireball.net/projects/markdown/syntax
+[Babelmark]: http://johnmacfarlane.net/babelmark2/
+[Using Pull Requests]: https://help.github.com/articles/using-pull-requests
+[action words]:
```

Comment:
This should point to https://help.github.com/articles/closing-issues-using-keywords/.